setup.py

```diff
@@ -395,9 +395,7 @@ def check_file_at_path(path2file):
         "numba.typed": ["py.typed"],
     },
     scripts=["numba/pycc/pycc", "bin/numba"],
-    author="Anaconda, Inc.",
-    author_email="numba-users@continuum.io",
-    url="https://numba.pydata.org",
+    url="https://numba.readthedocs.io/",
```

Comment:
Isn't https://numba.pydata.org Numba's home page? Is changing this to the docs page consistent with other packages?

Reply:
reverting